packages/editor/src/components/provider/index.js

```diff
@@ -194,8 +195,7 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 					editorSettings: getEditorSettings(),
 					isReady: __unstableIsEditorReady(),
 					mode: getRenderingMode(),
-					defaultMode:
-						postTypeObject?.default_rendering_mode ?? 'post-only',
+					defaultMode: postTypeObject?.default_rendering_mode,
```

Comment:
We need a fallback here in case the server provides a false value. Consumers can unset `default_rendering_mode` using PHP filters.

Reply:
Yes it was part of a debug change I included the missing commit